app/web/routers/default.py

```diff
@@ -11,22 +11,22 @@
 from app.web.security import get_user_state
 
 
-default_router = APIRouter()
+router = APIRouter()
```

Comment:
Since it is in the `default.py` file, the `default_router` name feels redundant. Each file contains its own specific context to prevent what feels like a stutter.